xwiki-platform-core/xwiki-platform-filter/xwiki-platform-filter-instance/xwiki-platform-filter-instance-api/src/main/java/org/xwiki/filter/instance/input/InstanceInputProperties.java

```diff
@@ -67,7 +67,7 @@ public boolean isVerbose()
     {
         Boolean verbose = (Boolean) get(PROPNAME_VERBOSE);
 
-        return verbose == Boolean.FALSE ? false : true;
+        return verbose != Boolean.FALSE;
```

Comment:
This looks like it should be the opposite, no?

Like:

```
return verbose;
```

Reply:
This is-
```
if(verbose == Boolean.FALSE) {
    return false;
} else { // verbose != Boolean.FALSE;
    return true;
}
```
so this is equivalent to return `verbose != Boolean.FALSE`, when verbose is not equal to Boolean.FALSE `true` is returned which is being done right now.

Reply:
if verbose is false then return false
if verbose is true then return true
==> return verbose

Reply:
Yes, you are right but I only simplified it to the point Checkstyle cannot detect it, Checkstyle does not detect `verbose != Boolean.FALSE`.

Reply:
But if you want, I can simplify it and inline the variable, would that be fine?

Reply:
> But if you want, I can simplify it and inline the variable, would that be fine?

I think that `return verbose;` is simpler to read and understand than `return verbose != Boolean.FALSE;`.

Thanks for your PR

Reply:
Actually `return get(PROPNAME_VERBOSE);` is even simpler.

Reply:
Done